loom/script/native/core/system/Platform/lmFile.cpp

```diff
@@ -148,7 +148,7 @@ class File
             return 1;
         }
 
-        utByteArray *ba = new utByteArray();
+        utByteArray *ba = lmNew(NULL) utByteArray();
```

Comment:
I assume since it's passed to script at the end `lualoom_pushnative<utByteArray>(L, ba);` the GC ends up lmDeleting it?